cmd/helm/installer/install.go

```diff
@@ -129,24 +129,23 @@ func generateLabels(labels map[string]string) map[string]string {
 }
 
 // parseNodeSelectors takes a comma delimited list of key=values pairs and returns a map
-func parseNodeSelectors(labels string) map[string]string {
+func parseNodeSelectors(labels string, m map[string]string) map[string]string {
```

Comment:
No, I wasn't suggesting that. If you want that function to mutate in place, then don't return a map as well. That eliminates the confusion. With that signature change, I'd consider renaming it to `parseNodeSelectorsInto`.